test/cases/compile_errors/directly_embedding_opaque_type_in_struct_and_union.zig

```diff
@@ -15,14 +15,8 @@ export fn b() void {
     _ = &bar;
 }
 export fn c() void {
-    const baz = &@as(O, undefined);
-    const qux = .{baz.*};
-    _ = qux;
-}
-export fn d() void {
-    const baz = &@as(O, undefined);
-    const qux = .{ .a = baz.* };
-    _ = qux;
```

Comment:
Don't lose this test coverage for `cannot load opaque type`. Perhaps add something like this:
```zig
export fn d() void {
    const ptr: *O = @ptrFromInt(0x1000);
    const x = .{ ptr.* };
    _ = x;
}
```

Reply:
done